server/package.json

```diff
@@ -71,20 +71,25 @@
     "nodemon": "^1.18.9",
     "nyc": "^11.9.0",
     "path": "^0.12.7",
-    "sequelize-cli": "^5.4.0",
+    "prettier": "^2.1.2",
+    "pretty-quick": "^3.1.0",
+    "sequelize-cli": "^5.5.1",
     "sinon": "^4.4.6",
     "sinon-chai": "^3.3.0",
     "supertest": "^3.4.2",
     "unionized": "^5.0.0"
   },
   "scripts": {
+    "format:check": "prettier --config ./.prettierrc --list-different \"src/{app,environments,assets}/**/*{.ts,.js,.json,.css,.scss}\"",
     "copy-frontend-assets": "./copy-frontend-assets.sh",
     "coverage": "nyc --check-coverage --lines 65 --reporter=html --reporter=text --reporter=lcov --extension .es6 npm test",
     "createdb": "sequelize db:create",
     "dropdb": "sequelize db:drop",
     "dev": "npm run copy-frontend-assets; npm run migrate; nodemon ./src/app.es6",
     "docs": "npm run jsdoc",
     "jsdoc": "jsdoc -c .jsdoc.json -d docs/code -t ./node_modules/ink-docstrap/template -r .",
+    "format:fix": "pretty-quick --staged",
+    "precommit": "run-s format:fix lint",
```

Comment:
These look like remnants from the `husky` implementation, if we don't need them let's take them out.

Reply:
@briandavidson - Same as my first comment. I removed this from the frontend but the server.... I went ahead and fixed that just now. Should be removed.